src/tribler/gui/widgets/downloadspage.py

```diff
@@ -239,13 +233,12 @@ def update_download_visibility(self):
             filter_match = self.window().downloads_filter_input.text().lower() in item.download_info["name"].lower()
             is_channel = item.download_info["channel_download"]
             if self.filter == DOWNLOADS_FILTER_CHANNELS:
-                item.setHidden(not is_channel or not filter_match)
+                ahide = not (is_channel and filter_match)
```

Comment:
**nitpick:** Is "ahide" correct English word?

Reply:
I've taken it from the definition:
```python
    def setHidden(self, ahide: bool) -> None: ...
```

Reply:
I suspect in the original C++ definition it was a prefix for "argument" to avoid name conflicts inside a function. Probably in Tribler code it is better to name the variable just "hide"

Reply:
Renamed